lib/easy_stalk/worker.rb

```diff
@@ -13,11 +13,15 @@ def work(job_classes = nil, on_fail: nil)
       job_classes = EasyStalk::Job.descendants if job_classes.compact.empty?
 
       job_classes.each do |job_class|
-        raise ArgumentError, "#{job_class} is not a valid EasyStalk::Job subclass" unless Class === job_class && job_class < EasyStalk::Job
+        if !job_class.is_a?(Class) || !(job_class < EasyStalk::Job)
```

Comment:
if you already have to parenthesize part it seems like it would be better to demorgan this and do

```
if ! (job_class.is_a?(Class) && job_class < EasyStalk::Job)
```

no?